node/src/chain_spec.rs

```diff
@@ -47,6 +47,10 @@ pub fn development_config() -> Result<ChainSpec, String> {
         "dev",
         ChainType::Development,
         move || {
+            let master_account_id: AccountId =
```

Comment:
How is it better for dev testnet rather than using built-in accounts such as Alice, Bob etc?